src/sorcha/ephemeris/simulation_constants.py

```diff
@@ -1,5 +1,7 @@
 import numpy as np
 
+GMSUN = 2.9591220828559115e-04
```

Comment:
I am little confused here. I thought in the current implementation, we calculate gmsun, but now it looks like it's hardcoded again

 https://github.com/dirac-institute/sorcha/blob/65e6362d9441128810ef211b5e52710a005286cb/src/sorcha/ephemeris/simulation_setup.py#L45

Reply:
Ah, I was just following what was in simulation_constants.py.  I can add in a calculation of GMTOTAL in simulation_setup.py, if needed.

Reply:
That would be cleaner if it's in simulation_setup.py. I think that also means changing the function calls to be parse_orbit_row(row, epoch, ephem, sun_dict, gm_sun, gm_total)

Reply:
That should be all set now.